### PR TITLE
ath79: add support for KuWfi CPE206

### PR DESCRIPTION
**Pull request by grosjo**
Short specification:
* CPU 650MHz / RAM 597MHz / AHB 216MHz
* Flash: 8 MB
* DRAM: 64 MB
* SoC: Qualcomm Atheros QCA9533 ver 2 rev 0
* Wifi 2.4G, IEEE 802.11b/g/n, 125mW / 21dBm, build in 12dBi MIMO panel antenna
* 1 Ethernet 1000Mbps port

Flash instruction under vendor firmware:

* connect the device to your computer via the POE provided
* set your computer IP to 192.168.1.120
* push the reset button while powering up the device
* connect to 192.168.1.20
* update bootloader to default bootloader using http://superwrt.com/download/files/bootloader/20180904_v1.3.10/firmware/superwrt_fboot_qca953x_20180904.img
* reboot, and power off.
* set your computer IP to 192.168.17.120
* again, push the reset button while powering up the device
* connect to 192.168.17.1
* use upgrade web interface to upload openwrt firmware
* device shall be booting on 192.168.1.1 as default

Signed-off by: Joan Moreau <jom@grosjo.net>

Thanks for your contribution to OpenWrt!

To help keep the codebase consistent and readable,
and to help people review your contribution,
we ask you to follow the rules you find in the wiki at this link
https://openwrt.org/submitting-patches

Please remove this message before posting the pull request.

### CODE REVIEW

File: target/linux/ath79/dts/qca9531_kuwfi_cpe206.dts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {  
+		compatible = "gpio-leds";
+
+		lan {
+			label = "cpe206:green:lan";
```
Comment (obsy) [inline target/linux/ath79/dts/qca9531_kuwfi_cpe206.dts:35]: drop cpe206 from label

File: target/linux/ath79/dts/qca9531_kuwfi_cpe206.dts [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		status: status {
+			label = "cpe206:green:status";
```
Comment (obsy) [inline target/linux/ath79/dts/qca9531_kuwfi_cpe206.dts:40]: drop cpe206 from label

File: target/linux/ath79/dts/qca9531_kuwfi_cpe206.dts [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+			default-state = "keep";
+		};
+
+		wlan {
+			label = "cpe206:green:wlan";
```
Comment (obsy) [inline target/linux/ath79/dts/qca9531_kuwfi_cpe206.dts:46]: drop cpe206 from label

**Review by obsy — CHANGES_REQUESTED** [review]

**Comment by achterin** [comment]
I might be wrong but your new pull-request looks identical to this one: https://github.com/openwrt/openwrt/pull/2506 and does not address the the review questions https://github.com/openwrt/openwrt/pull/2506#pullrequestreview-304321295

**Comment by hauke** [comment]
Please fix the formal issues:
````
=== Checking commit '0be7163f01bd6704de335113fd89a61ad70d72db'
Author name (Joan Moreau) seems ok
Commit subject line MUST start with '<area>: ' (Fixed 22.x)
Signed-off-by is missing or doesn't match author (should be 'Signed-off-by: Joan Moreau <jom@grosjo.net>')

A commit message exists
=== Checking commit '14c46a354e39b25730a346e93cd74f00ff2822be'
Author name (Joan Moreau) seems ok
Commit subject line seems ok (ath79: add support for KuWfi CPE206)
Signed-off-by is missing or doesn't match author (should be 'Signed-off-by: Joan Moreau <jom@grosjo.net>')
````

**Comment by Neustradamus** [comment]
@grosjo: Do not forget to fix this PR to have a good one and CPE206 support :)

Linked to:
- https://github.com/openwrt/openwrt/pull/2023
- https://github.com/openwrt/openwrt/pull/2506

**Pull request closed without merging**